### DISCUSSION TITLE
Log prob on Langfuse

### DISCUSSION THREAD

**Discussion by yusuke-intern** [langfuse/langfuse#2576] [Ideas]
### Describe the feature or potential improvement

It would be great if we can see log probability for outputs.
This helps preventing hallucinations.

 https://cookbook.openai.com/examples/using_logprobs

**Comment by marcklingen** [comment]
great idea, thank you!

**Comment by MJordahn** [comment]
Hi! Is there any update on this feature?

**Reply by marcklingen** [reply]
Not yet, open for contributions on this!

**Comment by Brandonn-Cruz** [comment]
Hi. Please tell me that this was already implemented :)
